ApplicationCode/GeoMech/GeoMechDataModel/RigFemPartResultCalculatorStressAnisotropy.cpp

```diff
@@ -130,3 +130,112 @@ RigFemScalarResultFrames* RigFemPartResultCalculatorStressAnisotropy::calculate(
     RigFemScalarResultFrames* requestedStress = m_resultCollection->findOrLoadScalarResult( partIndex, resVarAddr );
     return requestedStress;
 }
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+RigFemScalarResultFrames*
+    RigFemPartResultCalculatorStressAnisotropy::calculateTimeLapse( int partIndex, const RigFemResultAddress& resVarAddr )
+{
+    CVF_ASSERT( isMatching( resVarAddr ) );
+
+    caf::ProgressInfo frameCountProgress( m_resultCollection->frameCount() * 4, "" );
```

Comment:
We have to get used to using the better progress info system. The progress increments steal space in this method and makes it difficult to see the actual algorithmic code. I know the code is probably borrowed from similar code in the GeoMechExtractor, but even if that uses the old progress style, the new code should not use this. See:
https://ceetronsolutions.github.io/resinsight-system-doc/object/progress

Reply:
Fixed.